docs/requirements_for_rtd.txt

```diff
@@ -57,7 +57,7 @@ python-memcached==1.59
 python-mimeparse==1.6.0
 pytz==2018.9
 qe-tools==1.1.2
-reentry==1.2.2
+reentry>=1.3.0
```

Comment:
I am approving even though this might gives us problems at some point, given that we are breaking compatibility quite a bit in `reentry` minor versions. If we do, we will have to quickly release a patch of `aiida-core`